public/Get-DbaReplDistributor.ps1

```diff
@@ -59,12 +57,11 @@ function Get-DbaRepDistributor {
             # Connect to the distributor of the instance
             try {
                 $server = Connect-DbaInstance -SqlInstance $instance -SqlCredential $SqlCredential
```

Comment:
This should be a separate try/catch and use our standard Failure message

Reply:
This also seems redudant to call the Connection command. Why do we need to make two separate connections for the same object to be output?

The ComputerName, ServiceName and DomainInstanceName is already on the distributor object because `Get-DBaReplServer` returns those properties.